cli-commands/compiler/compiler.js

```diff
@@ -0,0 +1,28 @@
+const utils = require('./../util');
+
+let compiler = require("./etherlime-workflow-compile/index");
+let Resolver = require("./etherlime-resolver/index");
+
+const performCompilation = () => {
+    let resolverOptions = {
+        "working_directory": `${defaultPath}/contracts`,
```

Comment:
Where does defaultPath comes from? It should be a param but I do not see it passed